Example/StripeConnectExample/StripeConnectExample/Settings/Appearance/AppSettings+AppearanceInfo.swift

```diff
@@ -248,6 +251,110 @@ extension AppearanceInfo {
         appearance.typography.font = UIFont(name: "Handjet-Regular", size: UIFont.systemFontSize)
         return .init(displayName: "Custom Font", appearance: appearance)
     }
+
+    static var darkMode: AppearanceInfo {
+        var appearance = EmbeddedComponentManager.Appearance()
+
+        appearance.colors.primary = UIColor(red: 0/255, green: 133/255, blue: 255/255, alpha: 1)
```

Comment:
I'm a little surprised that this work as i would expect all of the divisions to be integer division. In any case, i see tons of prior art already in this file, so seems fine.